tablepyxl/style.py

```diff
@@ -172,8 +172,10 @@ def __init__(self, table):
         takes an html table object (from BeautifulSoup)
         """
         super(Table, self).__init__(table)
-        self.head = TableHead(table.thead, parent=self) if table.thead else None
-        self.body = TableBody(table.tbody or table, parent=self)
+        table_head = table.find('thead')
+        self.head = TableHead(table_head, parent=self) if table_head is not None else None
```

Comment:
I believe we still need the `if table_head is not None` check. `master` just has `if table.find('thead')`